2025-07-18 16:23:08,890 DEBUG    root            Loaded Command Group: ['gcloud', 'projects']
2025-07-18 16:23:08,896 DEBUG    root            Loaded Command Group: ['gcloud', 'projects', 'describe']
2025-07-18 16:23:08,899 DEBUG    root            Running [gcloud.projects.describe] with arguments: [--format: "value(projectNumber)", PROJECT_ID_OR_NUMBER: "formidable-bus-465107-j2"]
2025-07-18 16:23:08,946 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2025-07-18 16:23:08,982 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "GET /v1/projects/formidable-bus-465107-j2?alt=json HTTP/11" 403 None
2025-07-18 16:23:08,986 DEBUG    root            Chosen display Format:value(message.list(separator="
"))
2025-07-18 16:23:09,005 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2025-07-18 16:23:09,006 DEBUG    root            Chosen display Format:default
2025-07-18 16:23:09,009 DEBUG    root            Chosen display Format:value(message.list(separator="
"))
2025-07-18 16:23:09,010 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2025-07-18 16:23:09,010 DEBUG    root            Chosen display Format:default
2025-07-18 16:23:09,012 DEBUG    root            Chosen display Format:value(message.list(separator="
"))
2025-07-18 16:23:09,012 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2025-07-18 16:23:09,013 DEBUG    root            Chosen display Format:default
2025-07-18 16:23:09,017 DEBUG    root            Chosen display Format:value(message.list(separator="
"))
2025-07-18 16:23:09,017 DEBUG    root            Chosen display Format:value(links.flatten(show="values",separator="
"))
2025-07-18 16:23:09,018 DEBUG    root            Chosen display Format:default
2025-07-18 16:23:09,015 DEBUG    root            (gcloud.projects.describe) [504527767195-compute@developer.gserviceaccount.com] does not have permission to access projects instance [formidable-bus-465107-j2] (or it may not exist): Cloud Resource Manager API has not been used in project 504527767195 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. This command is authenticated as 504527767195-compute@developer.gserviceaccount.com which is the active account specified by the [core/account] property.
Cloud Resource Manager API has not been used in project 504527767195 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
Google developers console API activation
https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  domain: googleapis.com
  metadata:
    activationUrl: https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195
    consumer: projects/504527767195
    containerInfo: '504527767195'
    service: cloudresourcemanager.googleapis.com
    serviceTitle: Cloud Resource Manager API
  reason: SERVICE_DISABLED
Traceback (most recent call last):
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 985, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 945, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/surface/projects/describe.py", line 51, in Run
    return projects_api.Get(project_ref)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/api_lib/cloudresourcemanager/projects_api.py", line 129, in Get
    return client.projects.Get(
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/generated_clients/apis/cloudresourcemanager/v1/cloudresourcemanager_v1_client.py", line 758, in Get
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 744, in _RunMethod
    http_response = http_wrapper.MakeRequest(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/http_wrapper.py", line 359, in MakeRequest
    retry_func(ExceptionRetryArgs(http, http_request, e, retry,
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/http_wrapper.py", line 304, in HandleExceptionsAndRebuildHttpConnections
    raise retry_args.exc
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/http_wrapper.py", line 348, in MakeRequest
    return _MakeRequestNoRetry(
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/third_party/apitools/base/py/http_wrapper.py", line 406, in _MakeRequestNoRetry
    check_response_func(response)
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/api_lib/util/apis.py", line 231, in _CheckResponse
    _CheckForApiEnablementError(response_as_error)
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/api_lib/util/apis.py", line 197, in _CheckForApiEnablementError
    PromptToEnableApi(*enablement_info)
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/api_lib/util/apis.py", line 180, in PromptToEnableApi
    raise exception
googlecloudsdk.api_lib.util.exceptions.HttpException: [504527767195-compute@developer.gserviceaccount.com] does not have permission to access projects instance [formidable-bus-465107-j2] (or it may not exist): Cloud Resource Manager API has not been used in project 504527767195 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. This command is authenticated as 504527767195-compute@developer.gserviceaccount.com which is the active account specified by the [core/account] property.
Cloud Resource Manager API has not been used in project 504527767195 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
Google developers console API activation
https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  domain: googleapis.com
  metadata:
    activationUrl: https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195
    consumer: projects/504527767195
    containerInfo: '504527767195'
    service: cloudresourcemanager.googleapis.com
    serviceTitle: Cloud Resource Manager API
  reason: SERVICE_DISABLED
2025-07-18 16:23:09,020 ERROR    root            (gcloud.projects.describe) [504527767195-compute@developer.gserviceaccount.com] does not have permission to access projects instance [formidable-bus-465107-j2] (or it may not exist): Cloud Resource Manager API has not been used in project 504527767195 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. This command is authenticated as 504527767195-compute@developer.gserviceaccount.com which is the active account specified by the [core/account] property.
Cloud Resource Manager API has not been used in project 504527767195 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
Google developers console API activation
https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  domain: googleapis.com
  metadata:
    activationUrl: https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=504527767195
    consumer: projects/504527767195
    containerInfo: '504527767195'
    service: cloudresourcemanager.googleapis.com
    serviceTitle: Cloud Resource Manager API
  reason: SERVICE_DISABLED
